src/diffpy/pdfmorph/morphs/morphshape.py

```diff
@@ -165,15 +166,25 @@ def _spheroidalCF2(r, psize, axrat):
         f1 = (
             1
             - 3 * r / (4 * d * v) * (1 - r2 / (4 * d2) * (1 + 2.0 / (3 * v2)))
-            - 3 * r / (4 * d) * (1 - r2 / (4 * d2)) * v / sqrt(1 - v2) * atanh(sqrt(1 - v2))
+            - 3
```

Comment:
this line is probably more readable longer.  I would return it back and `# noqa` it I htink it is 501 code.